Documentation/QuickInstall/TheInstallTool/Index.rst

```diff
@@ -79,7 +79,28 @@ The Install Tool
       :class: with-shadow
 
       Install Tool in 1-2-3 mode, fifth step.
-
+      
+   **Introduction distribution**
```

Comment:
**Introduction Package**

    If you are using TYPO3 for the first time you may want to
    see a working example of the CMS before you start work on
    your own project.
    The `Official Introduction Package
    <https://extensions.typo3.org/extension/introduction/>`__ showcases many of TYPO3s
    capabilities and gives you the ability to try them first hand.
    The Introduction Package utilizes the `bootstrap_package extension
    <https://extensions.typo3.org/extension/bootstrap_package/>`__
    to generate multiple responsive HTML templates that you can select and try out.
    It also features examples of the different kinds page content that you typically see
    on a website. Such as paragraphs of text, images, tables and navigation menus.